about/reports/index.md

```diff
@@ -1,5 +1,10 @@
 # arXiv Reports
 
+## Annual Reports
+[2020 Annual Report](https://indd.adobe.com/view/fdd63397-e4b0-41af-b479-4845dc1ef48e)
```

Comment:
@afromme should we also add a PDF to download? This link may have some limitations for sharing.

Reply:
For now, we're going to leave it as a link. We may add a PDF version later.